### PR TITLE
Static b testpackage fix

### PR DESCRIPTION
**Pull request by ykempf**

### CODE REVIEW

**Comment by ykempf** [comment]
More diffs than expected, but "should be OK"?

**Comment by ykempf** [comment]
Test package is :heavy_check_mark: against the updated reference data which has the static B too.

File: .github/workflows/sync-hlrs-gitlab.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Generate testpackage reference data
```
[the remark this answers is not in the document]
Comment (markusbattarbee) [reply .github/workflows/sync-hlrs-gitlab.yml:1]: The HLRS sync script exists only in dev, not in master.

**Comment by ykempf** [comment]
Nah it's in dev and that's good enough for now.

**Pull request closed without merging**